instructions/2-lambda-function.md

```diff
@@ -14,13 +14,12 @@
 
 	```bash
 	$ cd lambda/custom
-	$ npm install
+	$ npm install --save
```

Comment:
the `save` option is not needed: did you find that you actually needed it?

It looks like the text above should be updated to remove the `--save` parameter from there.

Reply:
I didn't need it, you're right. We might want to update line 13 then to make the instructions consistent with the code sample.

Reply:
Can you update the pull request? Or would you prefer that we close it and I make the changes?